Dockerfile

```diff
@@ -8,5 +8,10 @@ ENV VERTICLE_HOME /usr/verticles
 # Copy your fat jar to the container
 COPY target/${VERTICLE_FILE} ${VERTICLE_HOME}/${VERTICLE_FILE}
 
+ENV LIB_DIR ${VERTICLE_HOME}/lib
+
+RUN mkdir -p ${LIB_DIR}
+COPY lib/* ${LIB_DIR}/
```

Comment:
Why isn't the `data-impor-processing-core-fat` library included in the jar for mod-inventory rather than externally?

Reply:
It's only for testing on the rancher, we must include these libraries because mod-inventory use several DataTypes. There will be a rollback before the merge